front/components/data_source/DocumentOrTableUploadOrEditModal.tsx

```diff
@@ -319,13 +365,7 @@ const DocumentUploadOrEditModal = ({
                       : plan.limits.dataSources.documents.sizeMb
                   } MB of raw text.`}
                   action={{
-                    label: uploading
-                      ? "Uploading..."
-                      : documentState.file
-                        ? documentState.file.name
-                        : initialId
-                          ? "Replace file"
-                          : "Upload file",
+                    label: creatingFile ? "Uploading..." : "Upload file",
```

Comment:
We lost showing wether it's replace or upload ?

Reply:
I also believe the button might become available while we are upserting (might be ok).

Reply:
This wasn't working on prod already so I dropped the 'replace' label for simplicity.
Good point about disabling it